src/state/models/content/profile.ts

```diff
@@ -61,6 +61,7 @@ export class ProfileModel {
   constructor(
     public rootStore: RootStoreModel,
     params: GetProfile.QueryParams,
+    withSetup: boolean = false, // withSetup is used when ProfileModel is used directly (not via another model like ProfileUiModel)
```

Comment:
I don't see this used elsewhere in this PR, what's it for?

Reply:
`// withSetup is used when ProfileModel is used directly (not via another model like ProfileUiModel)`

Reply:
Used here: https://github.com/bluesky-social/social-app/pull/1042/files#r1270009494

Reply:
🤦 sick thanks